sites/public/page_content/locale_overrides/general.json

```diff
@@ -15,5 +15,17 @@
     "headerLink": "City of Detroit - Housing and Revitalization Department",
     "forGeneralInquiries": "For general program inquiries, you may call the City of Detroit HRD at (313) 224-6380.",
     "forListingQuestions": "For listing and application questions, please contact the Property Agent displayed on each listing."
+  },
+  "eligibility": {
```

Comment:
Since this is a new feature, and not just overriding homepage content, is overrides the right place for the strings? Should these be in https://github.com/CityOfDetroit/bloom/blob/main/ui-components/src/locales/general.json?

Reply:
I was thinking about adding it there, but it feels a little bit off to add something in the ui-components directory which is implementation specific to Detroit. I know the separation isn't as clear in that the ui-components/src/locales/general.json file, because it already has application and Alameda county specific data. In my mind the ui-components directory should contain reusable ui-components only rather than site specific implementation details.

Reply:
It looks like other pages that are in sites/public have their strings in ui-components. For example, the `listings.closedListings` string, which is just on /listings, is [there](https://github.com/bloom-housing/bloom/blob/0b1cace2d75ef3d64a01f67a22185366ee9bc48d/ui-components/src/locales/general.json#L764).

So I agree that it feels weird to have them in ui-components since they're detroit specific, but imo it feels more weird not to follow the pattern, since this eligibility questionnaire is something that we could contribute upstream and other stuff in sites/public has their strings there.

Also, it seems like every string in `locale_overrides` also exists in the ui-components files, and that seems like a bad pattern to break, so maybe the best thing would be to put generic versions of all the questionnaire strings in ui-components and then add detroit specific overrides to the overrides file?

Reply:
Done.